content/plugins/descriptions/argo-cd.yaml

```diff
@@ -0,0 +1,94 @@
+# The server must be restarted to pick up changes in this file.
+
+# Required. Must be unique across all files in the directory. No whitespace. Use hyphens as
+# separators.
+name: argo-cd
+# Required. Can have whitespace. Should be titleized.
+humanName: Argo CD
+heading: 'Backstage Argo CD Plugin'
+# Keep it short
+lead: 'See Argo CD status in Backstage'
+
+seo:
+  # Don't forget to end with "| Roadie"
+  title: 'Backstage Argo CD Plugin | Roadie'
+  description: |
+    The Backstage Argo CD plugin integrates with your Argo CD instance api to show kubernetes status
+    information inside Backstage where it can be associated with your project.
+
+logo:
+  # This must be a relative path. It should start without a slash or with ./
+  # The overall size of the file should be approximately 200 by 200.
+  # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
+  # The image will be made greyscale by Gatsby Image Sharp.
+  fileSystemPath: './content/assets/logos/argo-cd/argo-cd-logo.png'
+  # The dimensions are required. They are the outer dimensions of the image file.
+  width: 350
+  height: 350
+
+coverImage:
+  # This must be a relative path. It should start without a slash or with ./
+  fileSystemPath: './content/assets/argo-cd-plugin.png'
+  alt: 'A preview of Argo CD overview widget including kubernetes pod status.'
+
+# Instructions for someone who wants to use this plugin.
+# languages used here must be listed in the .babelrc
+gettingStarted:
+  # What will this step accomplish?
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: yarn add @roadiehq/backstage-plugin-argo-cd
+  - intro: Add proxy config
+    language: yaml
+    code: |
+      // app-config.yaml
```

Comment:
Valid comments in yaml are `#` rather than double slashes.

Reply:
Done